src/Jwt/Symfony/Authentication/JsonWebToken.php

```diff
@@ -14,6 +14,11 @@
 
 class JsonWebToken extends AbstractToken
 {
+    public const TYPE_V1_JWT_PROVIDER_TOKEN = 'v1_jwt_provider_token';
```

Comment:
We need a `TYPE_` prefix? It also sounds good without the prefix 😊

Reply:
No not really, I'll remove it because it might make the places where it's used outside of this class a bit easier to read.

Reply:
Done in d23e862